src/charm.py

```diff
@@ -73,11 +79,25 @@ def __init__(self, framework: ops.Framework):
 
         self.container = self.unit.get_container("gocert")
         self.tls = TLSCertificatesProvidesV3(self, relationship_name="certificates")
+        self.dashboard = GrafanaDashboardProvider(self)
```

Comment:
Should we also pass `relation_name` to this integration? I know we use the default but we also use the default for the metrics integration

Reply:
dashboard moved to another pr